.yarnrc.yml

```diff
@@ -1,19 +1,7 @@
-nmHoistingLimits: workspaces
-
 nodeLinker: node-modules
 
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
     spec: "@yarnpkg/plugin-interactive-tools"
 
-yarnPath: .yarn/releases/yarn-3.5.1.cjs
-
-# Required to prevent "Error: Cannot find module '@storybook/react-webpack5/preset'"
```

Comment:
This was there due to a bug in Storybook that was fixed in 7.1.1 so now removed.